index.bs

```diff
@@ -2663,6 +2667,8 @@ during credential generation.
     ::  This value indicates that the [=[RP]=] wants to receive the [=attestation statement=] as generated by the
         [=authenticator=].
 
+    :   <dfn>enterprise</dfn>
+    ::  This value indicates that the [=[RP]=] wants to receive an [=attestation statement=] that may include uniquely identifying information. This is intended for controlled deployments within an enterprise where the organization wishes to tie registrations to specific authenticators. User agents MUST NOT provide such an attestation unless user agent or authenticator configuration permits it for the requested [=RP ID=]. If not permitted, the user agent SHOULD treat this value the same as `direct`.
```

Comment:
>If not permitted, the user agent SHOULD treat this value the same as `direct`.

Hm, could this "SHOULD" (as opposed to MUST) open opportunities for leaking the user's identity? If I put some known credential IDs in `excludeCredentials`, set `attestation: "enterprise"`, and the client refuses but does not treat it like `direct`...

If the client immediately returns failure, the RP knows that enterprise attestation was refused. If the client proceeds to the authenticator's test of user presence, the RP as usual does not learn if the authenticator does or does not have one of the `excludeCredentials`.

So probably not an issue?

Reply:
Have changed to a MUST because the SHOULD introduces unneeded ambiguity. I'm not actually sure whether fall back to &ldquo;direct&rdquo; or immediate rejection is actually the right behaviour. However, if we pick the former then we should firmly pick it.